### PR TITLE
Remove batches_update feature

### PR DESCRIPTION
**Pull request by GrapeBaBa**
Signed-off-by: grapebaba <281165273@qq.com>

## Motivation

Remove batches_update feature

## Related PRs

Fix issue #223

### CODE REVIEW

**Comment by GrapeBaBa** [comment]
@msmouse @sitalkedia

**Comment by msmouse** [comment]
@GrapeBaBa it's beautiful. Thank you so much.

Have you signed the CLA? https://github.com/aptos-labs/aptos-core/blob/main/documentation/contributing/individual-cla.pdf
If not, please sign and email a copy to us (address is in the doc).

**Comment by GrapeBaBa** [comment]
> @GrapeBaBa it's beautiful. Thank you so much.
> 
> Have you signed the CLA? https://github.com/aptos-labs/aptos-core/blob/main/documentation/contributing/individual-cla.pdf If not, please sign and email a copy to us (address is in the doc).

Cool. Let me do it.

**Comment by GrapeBaBa** [comment]
@msmouse how to rerun the build? I fixed lint error.

**Comment by msmouse** [comment]
@GrapeBaBa 

/canary

**Review by msmouse — COMMENTED** [review]
(sorry this comment didn't go through in the morning)

File: storage/scratchpad/src/sparse_merkle/node.rs [hunk 1]
```diff
@@ -113,13 +113,6 @@ impl<V> Node<V> {
         }
     }
 
-    pub fn new_internal(left: SubTree<V>, right: SubTree<V>, generation: u64) -> Self {
```
[the remark this answers is not in the document]
Comment (GrapeBaBa) [reply storage/scratchpad/src/sparse_merkle/node.rs:116]: fixed.

**Comment by GrapeBaBa** [comment]
> @GrapeBaBa: 🔑 Insufficient privileges: Not Collaborator

@msmouse Please help rerun the build.

**Comment by GrapeBaBa** [comment]
@msmouse @gregnazario Some test failed. Can be these tests ran in local machine? I didn't see it in development guide.

**Comment by msmouse** [comment]
@GrapeBaBa I think in this case it's just the test being flacky (We made these tests running against a locally built node in the CI process only recently.) But if you want, https://github.com/aptos-labs/aptos-core/blob/855013e0341e58d2fa942f0a3373adb26c62f84a/.circleci/config.yml#L223-L246 these are how they are ran.

/canary

**Comment by msmouse** [comment]
@GrapeBaBa  tests passing. Were you able to send the CLA?

**Comment by GrapeBaBa** [comment]
> @GrapeBaBa tests passing. Were you able to send the CLA?

just sent an email

**Review by msmouse — APPROVED** [review]

**Pull request closed without merging**